app/views/kiosk_reservations/_reservations_table.html.haml

```diff
@@ -10,7 +10,8 @@
       - reservations.each do |res|
         %tr
           %td
-            = res.user.full_name
+            - # Admin and Offline Reservations have a created_by user and no order user
+            = res.user&.full_name || res.created_by&.full_name || text("no_user")
```

Comment:
Can we put these into some methods on Reservation/AdminReservation/OfflineReservation?